### PR TITLE
Add update teams script for Fall 2018

### PR DESCRIPTION
**Pull request by J16053**

### CODE REVIEW

File: update-teams-fall-2018.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    "password": process.env.DATABASE_PASSWORD,
+    "database": "the_gazelle",
+    "charset": "latin1"
+  },
+});
+
+knex('semesters').insert([
+  {
+    'name': 'Fall 2018',
+    'date': '2018-09-22',
```
Comment (emilgoldsmith) [inline update-teams-fall-2018.js]: Can we just make this the first of september? As that was the start of the semester (I might also just change this myself)

**Review by emilgoldsmith — DISMISSED** [review]
Looks pretty good to me. You tested it locally right?

File: update-teams-fall-2018.js (hunk 1 above)
Comment (J16053) [reply update-teams-fall-2018.js]: Yes, it seemed to work locally without any issues!

**Review by emilgoldsmith — APPROVED** [review]
Tested it manually and seems to work great, thanks @J16053 ! I'll deploy it now

**Pull request merged**